generators/app/generator.mjs

```diff
@@ -30,7 +30,7 @@ export default class extends BaseApplicationGenerator {
 
   get [BaseApplicationGenerator.PROMPTING]() {
     return this.asPromptingTaskGroup({
-      async promptyForIonicDir() {
+      async promptForIonicDir() {
```

Comment:
@mshima I'm not sure if this is correct, but `prompty` seemed like a typo and I'm not prompted to type a directory when I run the following command:

```
jhipster jdl bug-tracker.jdl --blueprints ionic --skip-jhipster-dependencies
```